src/main/java/com/exadel/discount/controller/DiscountController.java

```diff
@@ -0,0 +1,45 @@
+package com.exadel.discount.controller;
+
+import com.exadel.discount.dto.DiscountDTO;
+import com.exadel.discount.dto.validation.Create;
+import com.exadel.discount.service.interfaces.DiscountService;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.UUID;
+
+@Validated
+@RestController
+@RequiredArgsConstructor
```

Comment:
Ad d this, please
@EqualsAndHashCode(exclude = {"tags", "coupons", "favorites"})
@ToString(exclude = {"tags", "coupons", "favorites"})

Reply:
No, will be add after yout PR